### DISCUSSION TITLE
Processor Architecture 64-bit in Catalina, unknown in Monterey and Ventura.

### DISCUSSION THREAD

**Discussion by MacDaddy1660B** [sickcodes/Docker-OSX#584] [General]
Hi,
It seems the Catalina VM works with 64-bit arch, but Monterey and Ventura have unknown processor types. Checking the Dockerfile shows they all use the Penryn type passed to qemu-system-x86_64. 

How is this?

**Comment by MacDaddy1660B** [comment]
> Hi, It seems the Catalina VM works with 64-bit arch, but Monterey and Ventura have unknown processor types. Checking the Dockerfile shows they all use the Penryn type passed to qemu-system-x86_64.
> 
> How is this?

Wait a minute. This seems to be a software issue I'm running into, and not a VM issue. Nevermind.

**Comment by camsmommy113** [comment]
> Hi,
> 
> It seems the Catalina VM works with 64-bit arch, but Monterey and Ventura have unknown processor types. Checking the Dockerfile shows they all use the Penryn type passed to qemu-system-x86_64. 
> 
> 
> 
> How is this?

**Comment by camsmommy113** [comment]
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Sunsets don&#39;t get much better than this one over <a href="https://twitter.com/GrandTetonNPS?ref_src=twsrc%5Etfw">@GrandTetonNPS</a>. <a href="https://twitter.com/hashtag/nature?src=hash&amp;ref_src=twsrc%5Etfw">#nature</a> <a href="https://twitter.com/hashtag/sunset?src=hash&amp;ref_src=twsrc%5Etfw">#sunset</a> <a href="http://t.co/YuKy2rcjyU">pic.twitter.com/YuKy2rcjyU</a></p>&mdash; US Department of the Interior (@Interior) <a href="https://twitter.com/Interior/status/463440424141459456?ref_src=twsrc%5Etfw">May 5, 2014</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
